src/server/system_services/pool_server.js

```diff
@@ -834,6 +834,15 @@ async function get_cloud_services_stats(req) {
     ]);
 
     const all_services = {};
+    for (const service of config.SERVICES_TYPES) {
+        all_services[service] = {
+            service: service,
```

Comment:
It's really optional, just making sure you know about this short syntax to set an object property from a variable that has the same name:
```suggestion
            service,
```